### ISSUE TITLE
Suggestions, workarounds and fixes for certain issues

### ISSUE THREAD

**Issue by HiMutz** [NecronomiconCoding/NecroBot#342]
I noticed when your pokemon bag is full you can't catch anymore pokemons
this could be fixed by a simple (while bag is full, turn on transfer temporarily)

also I noticed razzberry keeps increasing and take a huge space of the bag
option to set berries limit? if berriesCount > %limit%  recycle ?

option to set limits for pokeballs maybe? and prioritize collecting great/ultra balls over normal pokeballs 
for example:

max balls allowed (200)

if 200 pokeballs in your bag
and you collected 3 great balls
then you recycle 3 pokeballs and keep the great balls
for every great ball you add to your ball count, you delete a pokeball, till you have 200 great balls
same works with ultra, the idea is to collect the high tier balls

same thing with potions perhaps 

set item limit for revives, potions etc?

never come near lure modules, eggs etc they're precious xD

**Comment by Shroomzie** [comment]
spent two minutes looking at this repos code and then thought to comment eh?

**Comment by Shroomzie** [comment]
Waiting for you to push your code for this "workarounds and fixes for certain issues "

**Comment by HiMutz** [comment]
Kind of a douchey thing to say to a topic started with [suggestions]

**Comment by Shroomzie** [comment]
Is this a issue with the repo? Try Discord maybe!?

**Comment by HiMutz** [comment]
This is for the active contributors that might get inspired and enhance the logic of transfers, don't stress about it too much.

**Comment by Shroomzie** [comment]
Because they all hang out in the issues section of the repo and not the discord public dev channel?

**Comment by HiMutz** [comment]
we should probably ask github to remove the issues section because the discord public dev "chat" is a good place to keep track of things that needs to be fixed

**Comment by Shroomzie** [comment]
w/e kid, as long as you think trying to be heard amongst the noise of other issues flowing in a busy repo is a good idea then there is no changing your mind is there!

**Comment by HiMutz** [comment]
It's okay, I understand your dire need to call a stranger; a "kid" over the internet, I already knew the kind of douche you were when I read your first comment, by any means if you're here to argue, I refuse to be your punching bag. if this is not the right place to post what I posted, close it and move on, that of course if you have such authority and not just a wannabe. 

if you think what I wrote is useful then I've achieved my purpose. but I honestly see no need for you to go around posts unleashing the worst image of human personality upon mankind.

it's really that simple, post doesn't belong, remove it, post is useful? I'm glad.
